src/core/ext/filters/client_channel/client_channel.cc

```diff
@@ -277,8 +280,7 @@ class ChannelData {
   void UpdateStateAndPickerLocked(
       grpc_connectivity_state state, const absl::Status& status,
       const char* reason,
-      std::unique_ptr<LoadBalancingPolicy::SubchannelPicker> picker,
-      grpc_error* resolver_transient_failure_error = GRPC_ERROR_NONE);
+      std::unique_ptr<LoadBalancingPolicy::SubchannelPicker> picker);
 
   void UpdateServiceConfigInControlPlaneLocked(
```

Comment:
I guess technically this function name is ambiguous too.

Reply:
See above.